### PR TITLE
Website: Added links to blog posts

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
- Adds links to recent blog posts about instrumenting applications
- Improve image alt text

@annanay25 image links on this page are broken, but reviewing the code it looks just like all of our other image links.  any ideas?

https://grafana.com/docs/tempo/latest/guides/pushing-spans-with-http/

### CODE REVIEW

File: docs/tempo/website/guides/instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Instrumentation Examples
+---
+
+Check out these blog posts for help instrumenting your favorite languages to work with Tempo.
+
+- [Java](https://grafana.com/blog/2021/02/03/auto-instrumenting-a-java-spring-boot-application-for-traces-and-logs-using-opentelemetry-and-grafana-tempo/)
```
Comment (dgzlopes) [inline docs/tempo/website/guides/instrumentation.md:7]: (nit?) Perhaps we can specify that this tutorial uses Java auto-instrumentation.

**Review by dgzlopes — APPROVED** [review]

File: docs/tempo/website/guides/instrumentation.md (hunk 1 above)
Comment (joe-elliott) [reply docs/tempo/website/guides/instrumentation.md]: good call fixed.  going to leave open til tomorrow though b/c i want annanay to see the comment on the images

File: docs/tempo/website/guides/pushing-spans-with-http.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ docker run --env BACKEND=localhost:3100 --net host grafana/tempo-query:latest
 
 And open `http://localhost:16686/trace/0123456789abcdef` in the browser of your choice to see:
 
-<p align="center"><img src="pushing-spans-with-http.png" alt="Tempo Logo"></p>
+<p align="center"><img src="pushing-spans-with-http.png" alt="single span"></p>
```
Comment (annanay25) [inline docs/tempo/website/guides/pushing-spans-with-http.md:77]: When I right-click on the image https://grafana.com/docs/tempo/latest/guides/pushing-spans-with-http/#retrieving-traces here and select "Open in new tab" it takes me to https://grafana.com/docs/tempo/latest/guides/pushing-spans-with-http/pushing-spans-with-http.png

which means its looking for the image in a folder `pushing-spans-with-http`. If I change that link to "../pushing-spans-with-http.png" it works

Screenshot 2021-02-19 at 1 02 36 PM

Comment (joe-elliott) [reply docs/tempo/website/guides/pushing-spans-with-http.md]: good catch.  some of our links need the `../` and some don't apparently.  i do not have the time to figure out why :).  `../` added!

**Pull request merged**